src/playground-code-editor.ts

```diff
@@ -189,6 +201,13 @@ export class PlaygroundCodeEditor extends LitElement {
   @property({attribute: false})
   diagnostics?: Array<Diagnostic>;
 
+  @property({attribute: false})
+  completions?: EditorCompletion[];
+
```

Comment:
I think we should add a `showCompletions` setting to `code-editor` and `file-editor`, so that completions can be turned on and off.

Reply:
Should we have completions enabled by default, what do you think? If so, the attribute would need to be reversed due to it being a boolean attribute. 

In the case that we enable completions by default, I suggest the attribute would be something like `no-completions`.

Reply:
Hmm, yeah let's have them on by default. So I agree, a `no-completions` attribute / `noCompletions` property sounds right. It should be on `code-editor`, `file-editor`, and `ide`.